testing/run-atomic-tests.ps1

```diff
@@ -5,15 +5,18 @@ Install-AtomicRedTeam -getAtomics -verbose
 Import-Module "C:\AtomicRedTeam\invoke-atomicredteam\Invoke-AtomicRedTeam.psd1" -Force
 
 # Test Parameters
+$T1136001Args = @{ "password" = "Chiapet1" }
+$T1505003Args = @{ "web_shell_path" = "C:\inetpub\wwwroot", "web_shells" = "C:\AtomicRedTeam\atomics\T1505.003\src" }

```

Comment:
Codacy found an issue: [The assignment expression is not valid. The input to an assignment operator must be an object that is able to accept assignments, such as a variable or a property.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=5931858)